modules/integration/src/test/scala/scala/cli/integration/ExportJsonTestDefinitions.scala

```diff
@@ -0,0 +1,36 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+abstract class ExportJsonTestDefinitions(val scalaVersionOpt: Option[String])
+    extends ScalaCliSuite with TestScalaVersionArgs {
+  test("export json") {
+    val inputs = TestInputs(
+      os.rel / "Main.scala" ->
+        """//> using lib "com.lihaoyi::os-lib:0.7.8"
+          |
+          |object Main {
+          |  def main(args: Array[String]): Unit =
+          |    println("Hello")
+          |}
+          |""".stripMargin
+    )
+
+    inputs.fromRoot { root =>
+      os.proc(TestUtil.cli, "export", "--json", ".")
+        .call(cwd = root)
+
+      val fileContents = os.read(root / "dest" / "export.json")
+        .replaceAll("\\s", "")
+
+      expect(fileContents.contains("\"scalaVersion\":\"3.2.2\""))
```

Comment:
How large would be the full output? Maybe we could just compare the whole thing?

Reply:
Not very large, especially since there's one dependency in the file. We can do it, I'll change it.